Z2A SwiftUI/NSCoffee/NSCoffee/ContentView.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  ContentView.swift
+//  NSCoffee
+//
+//  Created by Rob Whitaker on 02/03/2025.
+//
+
+import SwiftUI
+
+struct ContentView: View {
+    private var drinks = Drinks()
+
+    var body: some View {
+        NavigationStack {
+            List {
+                Section("Coffees") {
```

Comment:
This will add the heading trait, won't it? I used regular cells on purpose. But in SwiftUI it might be way more unlikely that someone does that...

Reply:
Yeah, I found it difficult to find a way to make SwiftUI *not* add the header trait tbh. It really wants that trait in there.

Reply:
But I'll revisit this later I think.